diff --git a/debian/control b/debian/control
index f11b86fe..48220d7d 100644
--- a/debian/control
+++ b/debian/control
@@ -5,7 +5,7 @@ Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
 Build-Depends: debhelper-compat (= 13),
                dh-apport,
                dh-golang,
-               golang-go (>= 2:1.25~),
+               golang-1.25-go,
                apparmor,
                dbus,
                libdbus-1-dev,
diff --git a/debian/rules b/debian/rules
index f9705a3d..d7dc0b1c 100755
--- a/debian/rules
+++ b/debian/rules
@@ -31,2 +31,5 @@ export GOTOOLCHAIN := local

+# Run with Go 1.25
+export PATH := /usr/lib/go-1.25/bin/:$(PATH)
+
 %:
@@ -89,0 +90,2 @@ endif
+
+override_dh_dwz:
